cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -81,28 +85,83 @@ func (cmd *command) Run() error {
 	// initialize UI
 	var updateCh chan deployment.ProcessUpdate
 	if cmd.Verbose {
-		defer log.Println("Kyma deployed!")
+		//TODO: check for errors instead showing success message blindly - depends on https://github.com/kyma-incubator/hydroform/issues/170
+		defer cmd.showSuccessMessage()
 	} else {
 		asyncUI := asyncui.AsyncUI{StepFactory: &cmd.Factory}
 		updateCh, err = asyncUI.Start()
 		if err != nil {
 			return err
 		}
-		defer asyncUI.Stop() // stop receiving update-events and wait until UI rendering is finished
+		defer func() {
+			asyncUI.Stop() // stop receiving update-events and wait until UI rendering is finished
+			if !asyncUI.Failed {
+				cmd.showSuccessMessage()
+			}
+		}()
 	}
 
-	if cmd.opts.Domain == LocalKymaDevDomain { //patch Kubernetes DNS system when using "local.kyma.dev" as domain name
-		step := cmd.startDeploymentStep(updateCh, "Configure Kubernetes DNS to support Kyma local dev domain")
-		if err := ConfigureCoreDNS(cmd.K8s.Static()); err != nil {
-			return err
-		}
-		cmd.stopDeploymentStep(updateCh, step, (err == nil))
-		if err != nil {
-			return err
+	return cmd.deployKyma(updateCh)
+}
+
+func (cmd *command) showSuccessMessage() {
```

Comment:
This function will be removed when working on #702

Reply:
we probably have to add a lightweight template-engine for the summary, this will also include the success-message part.